File: Level 7kyu/Vowel Count.js
function getCount(str) {
    let vowelsCount = 0;
    let sCheck = "aeiou";

    for (let index = 0; index < str.length; index++) {
        if (sCheck.includes(str[index])) {
            vowelsCount++;
        }
    }
    return vowelsCount;
}

File: Level 7kyu/Vowel Count.rs
// Submitted
fn get_count(string: &str) -> usize {
    let mut vowels_count: usize = 0;
    let vowels_list = vec!['a', 'e', 'i', 'o', 'u'];
    for ch in string.chars() {
        if vowels_list.contains(&ch) {
            vowels_count += 1;        
        }
    }
    vowels_count
}
  
// Run to test
fn main() {
    let res = get_count("huynh viet tuan kiet");
    println!("{}", res);
}
